src/opnsense/service/templates/OPNsense/Dnsmasq/trust-anchors.conf

```diff
@@ -0,0 +1,6 @@
+# The root DNSSEC trust anchor, valid as at 11/01/2019
```

Comment:
I like to avoid templating static files and create them directly at the target location -- this way it gets removed if we do not need it

Reply:
@fichtner usually I would as well, but dnsmasq ships it already, but without the one we recently added.

Reply:
then this will flag in the health audit, yikes

Reply:
the last update of this file was 5 years ago

https://github.com/imp/dnsmasq/commits/master/trust-anchors.conf

I'm asking myself whether we're not trusting this file instead of fudging the anchors (I get why we did it from the command line). Now we don't need to if we trust them to update in time?

Reply:
I can live with both options, it just means we will revert https://github.com/opnsense/core/pull/7852

Reply:
I'm not attached to it, the old one probably doesn't expire in 2026 either

Reply:
ok, will trash this then. no problem, rather use defaults too, doubted a bit here.